components/layout_2020/fragment_tree/fragment.rs

```diff
@@ -30,7 +30,7 @@ pub(crate) enum Fragment {
     /// Floating content. A floated fragment is very similar to a normal
     /// [BoxFragment] but it isn't positioned using normal in block flow
     /// positioning rules (margin collapse, etc). Instead, they are laid out by
-    /// the [SequentialLayoutState] of their float containing block formatting
+    /// the [PositioningFragment] of their float containing block formatting
```

Comment:
`SequentialLayoutState` is correct here. I think you just need to import it.

Reply:
Hello @mrobinson . Thank you for this review. 
I have imported ```SequentialLayoutState``` like this; ```use crate::flow::float::SequentialLayoutState; ``` I get an error message; ```warning: unused import: `crate::flow::float::SequentialLayoutState`
  --> components/layout_2020/fragment_tree/fragment.rs:26:5
   |
26 | use crate::flow::float::SequentialLayoutState;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default ```
   
   I'm missing the correct way to import? Kindly advise.

Reply:
Sorry, I may have mislead you here. I would try to remove the import and use ``crate::flow::float::SequentialLayoutState` directly in the rustdoc comment. If that doesn't work, I would replace the brackets with backticks instead.